pkg/security/ImageScanService.go

```diff
@@ -115,9 +115,13 @@ func (impl *ImageScanServiceImpl) ScanImage(scanEvent *common.ImageScanEvent, to
 	}
 	wg := &sync.WaitGroup{}
 	wg.Add(1)
-	impl.ScanImageForTool(tool, executionHistory.Id, executionHistoryDirPath, wg, int32(scanEvent.UserId), ctx, imageScanRenderDto)
+	err = impl.ScanImageForTool(tool, executionHistory.Id, executionHistoryDirPath, wg, int32(scanEvent.UserId), ctx, imageScanRenderDto)
```

Comment:
add a TODO comment here that if multiple processes are to be done in parallel, then error propagation should have to be done via channels